src/views/projects/Patch.svelte

```diff
@@ -0,0 +1,372 @@
+<script lang="ts" context="module">
+  import type * as cobs from "@app/lib/cobs";
+  import type { Merge, Review } from "@app/lib/patch";
+
+  export interface TimelineReview {
+    inner: [string, Review];
+    type: "review";
+    timestamp: number;
+  }
+
+  export interface TimelineMerge {
+    inner: Merge;
+    type: "merge";
+    timestamp: number;
+  }
+
+  export interface TimelineThread {
+    inner: cobs.Thread;
+    type: "thread";
+    timestamp: number;
+  }
+</script>
+
+<script lang="ts">
+  import type { Project } from "@app/lib/project";
+
+  import * as router from "@app/lib/router";
+  import Authorship from "@app/components/Authorship.svelte";
+  import Badge from "@app/components/Badge.svelte";
+  import Changeset from "./SourceBrowser/Changeset.svelte";
+  import CobHeader from "@app/views/projects/Cob/CobHeader.svelte";
+  import CobSideInput from "./Cob/CobSideInput.svelte";
+  import Comment from "@app/components/Comment.svelte";
+  import CommitTeaser from "./Commit/CommitTeaser.svelte";
+  import Dropdown from "@app/components/Dropdown.svelte";
+  import HeaderToggleLabel from "./HeaderToggleLabel.svelte";
+  import TabBar from "@app/components/TabBar.svelte";
+  import Thread from "@app/components/Thread.svelte";
+  import { Patch } from "@app/lib/patch";
+  import { capitalize, formatCommit, isLocal } from "@app/lib/utils";
```

Comment:
There's no need to roll our own function, can just use the one in lodash:

```diff
diff --git a/src/components/TabBar.svelte b/src/components/TabBar.svelte
index e4c05264..6d9dd3bc 100644
--- a/src/components/TabBar.svelte
+++ b/src/components/TabBar.svelte
@@ -9,8 +9,8 @@
 <script lang="ts" strictEvents>
   type T = $$Generic;
 
+  import capitalize from "lodash/capitalize";
   import { createEventDispatcher } from "svelte";
-  import { capitalize } from "@app/lib/utils";
 
   export let options: Tab<T>[];
   export let active: T;
diff --git a/src/lib/utils.ts b/src/lib/utils.ts
index 81f50e87..e024fd39 100644
--- a/src/lib/utils.ts
+++ b/src/lib/utils.ts
@@ -114,11 +114,6 @@ export function formatCommit(oid: string): string {
   return oid.substring(0, 7);
 }
 
-export function capitalize(s: string): string {
-  if (s === "") return s;
-  return s[0].toUpperCase() + s.substring(1);
-}
-
 // Takes a path, eg. "../images/image.png", and a base from where to start resolving, e.g. "static/images/index.html".
 // Returns the resolved path.
 export function canonicalize(
diff --git a/src/views/projects/Issues.svelte b/src/views/projects/Issues.svelte
index f6c9ce3e..36b67cdb 100644
--- a/src/views/projects/Issues.svelte
+++ b/src/views/projects/Issues.svelte
@@ -11,7 +11,7 @@
 
   import * as router from "@app/lib/router";
   import * as utils from "@app/lib/utils";
-  import { capitalize } from "@app/lib/utils";
+  import capitalize from "lodash/capitalize";
   import { groupIssues } from "@app/lib/issue";
   import { sessionStore } from "@app/lib/session";
 
diff --git a/src/views/projects/Patch.svelte b/src/views/projects/Patch.svelte
index 91ca4be9..66750e06 100644
--- a/src/views/projects/Patch.svelte
+++ b/src/views/projects/Patch.svelte
@@ -36,9 +36,10 @@
   import HeaderToggleLabel from "./HeaderToggleLabel.svelte";
   import TabBar from "@app/components/TabBar.svelte";
   import Thread from "@app/components/Thread.svelte";
+  import capitalize from "lodash/capitalize";
   import { Patch } from "@app/lib/patch";
-  import { capitalize, formatCommit, isLocal } from "@app/lib/utils";
   import { createAddRemoveArrays } from "@app/lib/issue";
+  import { formatCommit, isLocal } from "@app/lib/utils";
   import { formatObjectId, validateTag } from "@app/lib/cobs";
   import { sessionStore } from "@app/lib/session";
 
diff --git a/src/views/projects/Patches.svelte b/src/views/projects/Patches.svelte
index 65f2cd5d..230e79f9 100644
--- a/src/views/projects/Patches.svelte
+++ b/src/views/projects/Patches.svelte
@@ -9,9 +9,9 @@
   import type { Project } from "@app/lib/project";
 
   import * as router from "@app/lib/router";
-  import * as utils from "@app/lib/utils";
   import PatchTeaser from "./Patch/PatchTeaser.svelte";
   import Placeholder from "@app/components/Placeholder.svelte";
+  import capitalize from "lodash/capitalize";
 
   export let patches: Patch[];
   export let status: PatchStatus;
@@ -59,7 +59,7 @@
     </div>
   {:else}
     <Placeholder emoji="🍂">
-      <div slot="title">{utils.capitalize(status)} patches</div>
+      <div slot="title">{capitalize(status)} patches</div>
       <div slot="body">No issues matched the current filter</div>
     </Placeholder>
   {/if}

```

Reply:
can we do that in another PR?

Reply:
Sure.